modules/oe_subscriptions_anonymous/src/Form/SettingsForm.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_subscriptions_anonymous\Form;
+
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Entity\Element\EntityAutocomplete;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * The form for the configuration of the Anonymous subscription module.
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * Name of the config being edited.
+   */
+  const CONFIG_NAME = 'oe_subscriptions_anonymous.settings';
+
+  /**
+   * EntityTypeManager.
+   *
+   * @var Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'oe_subscriptions_anonymous_settings_form';
+  }
+
+  /**
+   * Constructs a settings form.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   The entity manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager) {
+    $this->entityTypeManager = $entityTypeManager;
```

Comment:
No call to parent? No constructor property promotion?

Reply:
Added